### PR TITLE
don't send CLUSTER_ID to segment for config subcommand and just 'dcos'

### PR DESCRIPTION
**Pull request by tamarrow**

### CODE REVIEW

**Comment by jsancio** [comment]
We already do this: https://github.com/mesosphere/dcos-cli/blob/master/cli/dcoscli/main.py#L42-L44

Can we make sure that we are adding this check in the right place?

**Comment by tamarrow** [comment]
that just make sure the user is registered, but doesn't stop us reporting. that's why i added tests!

File: cli/dcoscli/analytics.py [hunk 1]
```diff
@@ -261,11 +262,12 @@ def _base_properties(conf=None):
...
-        cluster_id = mesos.DCOSClient().metadata().get('CLUSTER_ID')
-    except:
-        logger.exception('Unable to get the cluster_id of the cluster.')
-        cluster_id = None
+    cluster_id = None
+    if command and command != "config":
+        try:
+            cluster_id = mesos.DCOSClient().metadata().get('CLUSTER_ID')
+        except:
+            logger.exception('Unable to get the cluster_id of the cluster.')
```
Comment (jsancio) [inline cli/dcoscli/analytics.py]: Let's log the exception too.

**Comment by jsancio** [comment]
:+1: :+1: LGTM. One minor comment!

**Pull request merged**